.gitignore

```diff
@@ -21,3 +22,5 @@ coverage_summary.txt
 
 # Ignore external libraries
 deps/
+runtime/src/paratest_genesis
+runtime/src/paratest_wasm
```

Comment:
If I saw well you've removed the migration parachain logic...Maybe we can remove this too.